### ISSUE TITLE
Cannot upgrade to the latest version because PHP is not compatible.

### ISSUE THREAD

**Issue by mtuma5672** [grokability/snipe-it#15764]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

I tried upgrading Snipe-it after upgrading to the latest PHP (8.2.1). The release notes state "Snipe-IT now requires PHP 8.1.2 or greater". However when I try to upgrade, I get this message: "This version of PHP (8.2.10-2ubuntu2.2) is NOT compatible with Snipe-IT. Snipe-IT requires PHP versions between 7.4.0 and 8.2.0."

Do I need to downgrade to version 8.2.0??

### Reproduction steps

1. Upgrade Ubuntu.
2. Upgrade Snipe-IT.

### Expected behavior

The upgrade progresses because I have met the requirements.

### Snipe-IT Version

v6.1.2 build 10938 (g32747cafd)

### Operating System

Ubuntu

### Web Server

Apache

### PHP Version

8.2.10-2ubuntu2.2

**Comment by snipe** [comment]
I'd say maybe try doing a `git pull` first, and then running the upgrader again.

https://github.com/snipe/snipe-it/blob/master/.upgrade_requirements.json


<blockquote><img src="https://repository-images.githubusercontent.com/14346979/164a6580-7d79-11e9-865f-eae87593e11a" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/snipe/snipe-it/blob/master/.upgrade_requirements.json">snipe-it/.upgrade_requirements.json at master · snipe/snipe-it</a></strong></div><div>A free open source IT asset/license management system - snipe/snipe-it</div></blockquote>

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)

**Comment by JohnBlunden** [comment]
I found this thread after googling. I have about the same issue. I have version 8.3.7 of SnipeIT and it was installed quite recently and is working fine, but I want to upgrade to 8.4.0 that is available, but after running php upgrade.php i get this error:

> Launching using branch: master
> Found PHP requirements, will check for PHP > 8.2.0 and < 8.5.0
> 
> !!!!!!!!!!!!!!!!!!!!!!!!! PHP VERSION ERROR !!!!!!!!!!!!!!!!!!!!!!!!!
> This version of PHP (8.5.2) is NOT compatible with Snipe-IT.
> Snipe-IT requires PHP versions between 8.2.0 and 8.5.0.

I tried git pull, but it does not help.
Is there some way around this to do the upgrade or do I have to downgrade php even though it works just fine with version 8.3.7, but not working for a newer version?
